code/_onclick/other_mobs.dm

```diff
@@ -114,8 +120,13 @@
 /*
 	Animals & All Unspecified
 */
+
+// If the UnarmedAttack chain is blocked
+#define LIVING_UNARMED_ATTACK_BLOCKED (HAS_TRAIT(src, TRAIT_HANDS_BLOCKED) \
+		|| SEND_SIGNAL(src, COMSIG_LIVING_UNARMED_ATTACK, A, proximity_flag, modifiers) & COMPONENT_CANCEL_ATTACK_CHAIN)
```

Comment:
One indent.

Reply:
Also, make this macro take `A` as an argument, so that we can change the variable names later without much problem (it's also just bad form to act unhygienically).